src/drawers/symbolDrawer.ts

```diff
@@ -56,7 +56,7 @@ export function makeSymbolCanvasDrawStep(
 
             // check attributes and symbol type
             const attrsSame = isAttributeValuesEqual(prevAttrs, attrs, ContextStyleAttrs);
-            const symbolGenerator = symbolAccessor(datum, index, this._dataset);
+            const symbolGenerator = symbolAccessor(datum, index, dataset);
```

Comment:
I wonder why this ever worked, since the `this` in a browser context refers to the `window` object. Maybe a transpilation bug that is surfacing now that we've upgraded typescript?